WeaveApp/webpack.common.js

```diff
@@ -0,0 +1,112 @@
+const path = require('path');
```

Comment:
This is all the common webpack config required for either a development or production build.